CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+*.sh jonathan-enf
+/scripts jonathan-enf
```

Comment:
you won't be able to merge code in with just you as a CODEOWNER? github requires a codeowner that is NOT the author of the code. Add at least another person.

Reply:
Done.